app/views/step_by_step_pages/index.html.erb

```diff
@@ -51,35 +51,7 @@
             name: "status",
             label: "",
             full_width: true,
-            options: [
-              {
-                text: "All",
-                data_attributes: {
-                  show: "all"
-                },
-              },
-              {
-                text: "Draft",
-                value: "draft",
-                data_attributes: {
-                  show: "draft"
-                },
-              },
-              {
-                text: "Published",
-                value: "published",
-                data_attributes: {
-                  show: "published"
-                },
-              },
-              {
-                text: "Scheduled",
-                value: "scheduled",
-                data_attributes: {
-                  show: "scheduled"
-                },
-              }
-            ].each{ |item| item[:selected] = item[:value] == params[:status] }
+            options: StepByStepFilter::Options.statuses.each{ |item| item[:selected] = item[:value] == params[:status] }
```

Comment:
Nice!

Think it should have a space between `each` and `{`.

Reply:
and what moving this to a method in `StepByStepFilter::Options` to keep the view cleaner?

Reply:
@erino That's an interesting idea. I'll investigate how to do that.

Reply:
@erino Refactored and fixed-up